src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -497,30 +505,20 @@ public void canCheckOutViaProxy() {
   }
 
   @Test
-  public void cannotCheckOutWhenLoanPolicyDoesNotExist() {
+  public void cannotCreateCirculationRuleWhenLoanPolicyDoesNotExist() {
```

Comment:
Has this now test moved to the circulation rules API tests?

Reply:
moved